src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleasePage.tsx

```diff
@@ -127,15 +127,15 @@ class PublicationReleasePage extends Component<Props> {
                   </li>
                 ))}
               </ul>
-              <h2 className="govuk-heading-m govuk-!-margin-top-9">
+              {/* <h2 className="govuk-heading-m govuk-!-margin-top-9">
```

Comment:
Code can be removed rather than commented out :)

Reply:
Done :)